ui/pages/Project/utils/histogramSelectors.js

```diff
@@ -1,47 +0,0 @@
-import { createSelector } from 'reselect'
-
-import { getIndividualsByGuid } from 'redux/utils/commonDataActionsAndSelectors'
-import { getNameForCategoryHpoId } from 'shared/utils/hpoUtils'
-import { getFamilyGuidToIndividuals } from '../utils/visibleFamiliesSelector'
-
-
-/**
- * function that returns a mapping of familySize => number of families
- *
- * @param state {object} global Redux state
- */
-export const getFamilySizeHistogram = createSelector(
-  getFamilyGuidToIndividuals,
-  (familyGuidToIndividuals) => {
-    const familySizes = Object.values(familyGuidToIndividuals)
-      .map(individualsInFamily => Math.min(individualsInFamily.length, 5))
```

Comment:
@bw the logic for familySizeHistogram came directly from here, which had the limit of 5

Reply:
🤦‍♂️